docs/docsite/rst/userguide/inventories.rst

```diff
@@ -481,7 +481,7 @@ Inventory updates use dynamically-generated YAML files which are parsed by their
 - :ref:`ug_source_openstack`
 - :ref:`ug_source_rhv`
 - :ref:`ug_source_rhaap`
-- :ref:`ug_source_terraform`
```

Comment:
why was this removed?

Reply:
Good catch, I must have accidentally typed over it instead of copying an entry for the new one.